src/main/java/frc/robot/NetworkTables/NetworkTablesBase.java

```diff
@@ -8,7 +8,7 @@
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 //Class
-public class NetworkTablesBase {
+public class NetworkTablesBase extends Thread {
```

Comment:
To properly extend the `Thread` class, add the `@Override` annotation to the `run()` method and get it out of the constructor.